### PR TITLE
Multiple commands from one plugin

### PR DESCRIPTION
**Pull request by theVDude**
Tested with a couple of plugins I made, disable and enable work as expected. You can disable one command from a plugin with multiple and the others will work.

### CODE REVIEW

File: plugins [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 3f15be9b0eb82c126c083295505e8d6dc3da6df6
+Subproject commit fd7e8804edb5463a0515c2ad161ac7d11e3ef3ca
```
Comment (sigv) [inline plugins:1]: May I ask why the plugin repository is being pushed back?

**Comment by sigv** [comment]
I have made pull requests for the plugins so that they can be updated when this pull request gets approved into master by totallymike.

File: plugins (hunk 1 above)
Comment (theVDude) [inline plugins:1]: I'm not positive, I suck at git.

**Comment by sigv** [comment]
Pull requests [#2](https://github.com/totallymike/ircnode_plugins/pull/2) and [#3](https://github.com/totallymike/ircnode_plugins/pull/3) in ircnode_plugins repo should be merged into master as they implement the changes done in this pull request.

**Pull request merged**